lib/matplotlib/axes/_axes.py

```diff
@@ -4838,11 +4839,23 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
             - If a sequence of values, the values of the lower bound of
               the bins to be used.
 
-        xscale : {'linear', 'log'}, default: 'linear'
-            Use a linear or log10 scale on the horizontal axis.
+        xscale : {'linear', 'log', 'symlog'}, default: 'linear'
+            The scale used for the horizontal axis.
 
-        yscale : {'linear', 'log'}, default: 'linear'
-            Use a linear or log10 scale on the vertical axis.
+            'symlog' creates a `.SymmetricalLogScale`, which has optional
+            parameters. If you want to modify them, pass a tuple
+            ``('symlog', params_dict)`` instead of the plain string 'symlog';
+            e.g.
+            ``hexbin(..., xscale=('symlog', {'base': 2, 'linthresh': 3}))``.
+
+        yscale : {'linear', 'log', 'symlog'}, default: 'linear'
+            The scale used for the vertical axis.
+
+            'symlog' creates a `.SymmetricalLogScale`, which has optional
+            parameters. If you want to modify them, pass a tuple
+            ``('symlog', params_dict)`` instead of the plain string 'symlog';
+            e.g.
+            ``hexbin(..., yscale=('symlog', {'base': 2, 'linthresh': 3}))``.
```

Comment:
Can merge:
```suggestion
        xscale, yscale : {'linear', 'log', 'symlog'}, default: 'linear'
            The scale used for the horizontal / vertical axis.

            'symlog' creates a `.SymmetricalLogScale`, which has optional
            parameters. If you want to modify them, pass a tuple
            ``('symlog', params_dict)`` instead of the plain string 'symlog';
            e.g.::
            
                hexbin(..., xscale=('symlog', {'base': 2, 'linthresh': 3}))
```